addons/l10n_din5008/data/report_layout.xml

```diff
@@ -7,5 +7,9 @@
             <field name="pdf"></field>
             <field name="name">DIN 5008</field>
         </record>
+        <record forcecreate="True" id="display_position_column" model="ir.config_parameter">
```

Comment:
Why `forcecreate="True"`?
I would have this `forcecreate="False"` and in a `noupdate="True"` block to avoid anything changing when updating the module (or upgrading Odoo)